qdev_wrappers/station_configurator.py

```diff
@@ -0,0 +1,153 @@
+from contextlib import suppress
+from typing import Optional
+import importlib
+import logging
+import yaml
+
+import qcodes
+from qcodes.instrument.base import Instrument
+from qcodes.station import Station
+import qcodes.utils.validators as validators
+from qcodes.instrument.parameter import Parameter
+from qcodes.monitor.monitor import Monitor
+from .parameters import DelegateParameter
+
+log = logging.getLogger(__name__)
+
+# config from the qcodesrc.json file (working directory, home or qcodes dir)
+auto_reconnect_instrument = qcodes.config["user"]["auto_reconnect_instrument"]
+
+
```

Comment:
Should the atexit register logic be added to the station Configurator? Since it creates the instruments I think so

Reply:
I would rather prefer not to. The way it is now the StationConfigurator is an optional component, that you can use if you like it. The atexit should not be optional. But I have to say that I don't have a better idea where to put it.
What about the init of qdev wrappers. It might get added multiple times then but this should not break anything. Now that we are excluding packages from the UMR this should work, right?

Reply:
Yes running it multiple times should be fine. We still need to merge https://github.com/QCoDeS/Qcodes/pull/1009 to fix the UMR issue